frontend/pages/hosts/details/cards/Software/InstallStatusCell/InstallStatusCell.tsx

```diff
@@ -458,6 +458,7 @@ const InstallStatusCell = ({
           commandUuid: (lastInstall as IAppLastInstall).command_uuid,
         }),
       });
+      return;
```

Comment:
Exit early to avoid calling `onShowInstallDetails`
This causes two modals to pop up
<img width="1439" height="637" alt="Screenshot 2025-12-10 at 1 48 54 PM" src="https://github.com/user-attachments/assets/a6d4b5d0-3a35-4ae3-9dea-86882e612129" />